pkg/cmd/dedicated/register/registercluster.go

```diff
@@ -373,7 +373,7 @@ func registerClusterWithKasFleetManager(opts *options) error {
 	if err != nil {
 		return err
 	}
-	err = clustermgmt.CreateAddonWithParams(opts.f, opts.clusterManagementApiUrl, opts.accessToken, getStrimziAddonIdByEnv(con), response.FleetshardParameters, opts.selectedCluster.ID())
+	err = clustermgmt.CreateAddonWithParams(opts.f, opts.clusterManagementApiUrl, opts.accessToken, getStrimziAddonIdByEnv(con), nil, opts.selectedCluster.ID())
```

Comment:
hmm this was done in https://github.com/redhat-developer/app-services-cli/pull/1815 ? A rebase maybe ;-)

Reply:
Yes, it was. Great catch 👍